scalalib/src/mill/scalalib/JavaModule.scala

```diff
@@ -92,6 +95,95 @@ trait JavaModule
           case _: ClassNotFoundException => // if we can't find the classes, we certainly are not in a ScalaJSModule
         }
     }
+
+    private def quickTest(args: Seq[String]): Task[(String, Seq[TestResult])] =
+      Task(persistent = true) {
```

Comment:
`Task(persistent = true)`  is probably the wrong thing here, since `def quickTest` takes arguments. We should add a `Command(persistent = true)` flag and keep it as a command similar to the other test tasks